docs/pages/management/operations/ca-rotation.mdx

```diff
@@ -63,7 +70,7 @@ Administrators can abort the rotation and revert all changes any time before
 the rotation is completed by entering the `rollback` phase.
 
 ```sh
-$ tctl auth rotate --phase=rollback --manual
+$ tctl auth rotate --phase=rollback --type={host|user|db|openssh|jwt|all} --manual
```

Comment:
Nit: The curly brace notation for listing possible flag values only seems to exist on this page:

```
$ grep -RlE "\{([a-z]+\|)+[a-z]+\}" docs/pages
docs/pages/management/operations/ca-rotation.mdx
```
Is there a notation we can use that's more consistent with the rest of the docs?

Reply:
That is the common notation that is used for describing the cli options but I looked around at other parts and it looks like we also use an input box so I replaced it with that to be more consistent.